tests/test_build_html.py

```diff
@@ -147,7 +147,7 @@ def check_extra_entries(outdir):
 @pytest.mark.sphinx('html', testroot='warnings')
 def test_html_warnings(app, warning):
     app.build()
-    html_warnings = strip_escseq(warning.getvalue().replace(os.sep, '/'))
+    html_warnings = strip_escseq(re.sub(re.escape(os.sep) + '{1,2}', '/', warning.getvalue()))
```

Comment:
It makes the replacement treat one or two `\` as the same thing, otherwise `\\` patterns get turned into `//` which doesn't work right.